core/unit_test/TestViewMemoryAccessViolation.hpp

```diff
@@ -128,6 +131,8 @@ void test_view_memory_access_violations_from_device() {
   test_view_memory_access_violation(make_view<V6>(lbl), exec_space, prefix + ".*UNAVAILABLE");
   test_view_memory_access_violation(make_view<V7>(lbl), exec_space, prefix + ".*UNAVAILABLE");
   test_view_memory_access_violation(make_view<V8>(lbl), exec_space, prefix + ".*UNAVAILABLE");
+ // We check the memory space for unmanaged Views when configuring with Kokkos_ENABLE_DEBUG
+#ifndef KOKKOS_ENABLE_DEBUG
```

Comment:
Same here.  What is wrong?  The construction is allowed and tested in `TestViewEmptyRuntimeUnmanaged.hpp`.
Why wouldn't we get to the memory access violation in debug mode?